### PR TITLE
Add new github orgs to the fetch-repos.py script used by codesearch container

### PR DESCRIPTION
**Pull request by SohamChakraborty**
* Part of [2182](https://github.com/kubernetes/k8s.io/issues/2182)
* Add remaining github organizations to the script

### CODE REVIEW

File: images/codesearch/cs-fetch-repos/src/fetch-repos.py [hunk 1]
```diff
@@ -27,6 +27,9 @@
     'kubernetes-csi',
     'kubernetes-incubator',
     'kubernetes-sigs',
+    'kubernetes-nightly',
```
Comment (dims) [inline images/codesearch/cs-fetch-repos/src/fetch-repos.py:30]: please don't add the nightly and retired orgs!

Comment (SohamChakraborty) [reply images/codesearch/cs-fetch-repos/src/fetch-repos.py]: Understood, removed them now :)

File: images/codesearch/cs-fetch-repos/src/fetch-repos.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright 2021 The Kubernetes Authors.
+# Copyright 2024 The Kubernetes Authors.
```
Comment (ameukam) [inline images/codesearch/cs-fetch-repos/src/fetch-repos.py:3]: No need to change this.

Comment (SohamChakraborty) [reply images/codesearch/cs-fetch-repos/src/fetch-repos.py]: Changed it back.

**Pull request merged**